spec/helpers/activity_helper_spec.rb

```diff
@@ -59,19 +60,20 @@
       end
     end
 
-    context "when the activity has passed the country step" do
+    context "when the activity has passed the region step" do
```

Comment:
Do we need the equivalent for `country`, too?

Reply:
I don't think we want to test every step here, just test that the step logic works - is this another place where country was refactored to region?